lib/breakers/statsd_plugin.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module Breakers
-  class StatsdPlugin
+  class StatsDPlugin
```

Comment:
Crazy nitpick but, by convention, wouldn't the filename become `stats_d_plugin.rb`?

Reply:
@kfrz would it make sense to inflect it?

Reply:
Ooh, yes & yes ! Thanks gents, great suggestions.